app/client/src/components/shared/LinkButton.js

```diff
@@ -15,8 +15,9 @@ const linkButtonStyles = css`
 
   &:hover,
   &:focus {
+    background-color: inherit !important;
+    color: inherit !important;
```

Comment:
Similar to above, this is causing the text to be black on hover and focus. It looks like link buttons previously turned purple on hover, and other links use navy blue. I think the latter should be used for consistency internally and with USWDS:
```suggestion
    color: ${colors.navyBlue()} !important;
```